ktorm-support-mysql/src/test/kotlin/me/liuwj/ktorm/support/mysql/MySqlTest.kt

```diff
@@ -65,6 +65,10 @@ class MySqlTest : BaseTest() {
                 it.salary to 100
                 it.departmentId to 2
             }
+            onDuplicateKey {
+                it.name to it.name
+                it.job values it.job
```

Comment:
Would it be better to change the syntax to `it.job to values(it.job)`?

Maybe we can also support `it.c to (values(it.a) + values(it.b))`, just like the examples here: https://mariadb.com/kb/en/values-value/. It is easy to do that by just adding a `values` function like: 

```kotlin
fun <T : Any> values(expr: ColumnDeclaring<T>): FunctionExpression<T> {
    return FunctionExpression(
        functionName = "values",
        arguments = listOf(expr.asExpression()),
        sqlType = expr.sqlType
    )
}
```

Reply:
@vincentlauvlwj Indeed, it looks more easy to understand.
I updated code.